GlimpseImputationPipeline/Glimpse2Imputation.wdl

```diff
@@ -80,8 +85,8 @@ workflow Glimpse2Imputation {
                 fasta_index = fasta_index,
                 preemptible = preemptible,
                 docker = docker,
-                cpu = select_first([safety_check_n_cpu, SelectResourceParameters.request_n_cpus]),
-                mem_gb = select_first([safety_check_memory_gb, SelectResourceParameters.memory_gb]),
+                cpu = select_first([cpu_phase, safety_check_n_cpu, SelectResourceParameters.request_n_cpus]),
```

Comment:
Shouldn't the safety checks go first? Maybe I misunderstand what the purpose of the phase variables are.

Reply:
`safety_check_n_cpu` was intended since the parameters were being set algorithmically, so we don't want the user to be surprised by a request for a ton of resources.  My feeling with an input variable though is that the user shouldn't be surprised, since they have set the value in the first place.  So there's no need for the safety check in that scenario.  I could also see a scenario where for some reason we might want to push the resources higher than the safety check manually, and the wdl should allow for that.

Also, even without the safety check here, there is always the safety checks of quota that won't allow someone to ask for like a petabyte of ram.

Reply:
I see, that make sense, thanks